packages/dds/tree/src/simple-tree/api/simpleSchema.ts

```diff
@@ -125,19 +125,11 @@ export interface SimpleFieldSchema {
  *
  * @sealed
  */
-export interface SimpleTreeSchema {
+export interface SimpleTreeSchema extends SimpleFieldSchema {
```

Comment:
This gives `SimpleTreeSchema` the new `kind` property... Does it conceptually mean the kind for the root field for the tree? Related, can `SimpleTreeSchema` represent subtrees (and in that case, `kind` means the kind of the field in the parent node, which acts as the root field for the subtree)?

Reply:
> This gives SimpleTreeSchema the new kind property... Does it conceptually mean the kind for the root field for the tree?

Yes. This was a gap in the original implementation. The root field of a tree may be optional (before, this code assumed it was always required). Fundamentally, the root of the tree is a field, not a node. The code previously was structured around the root input being a node (which it then mapped to an implicitly required field). This change makes this representation more consistent with the SharedTree's underlying representation.

> Related, can SimpleTreeSchema represent subtrees (and in that case, kind means the kind of the field in the parent node, which acts as the root field for the subtree)?

Yeah, basically. Really, it makes the root less special, so it's like any other field location in the tree.